_includes/selector.html

```diff
@@ -556,14 +556,14 @@
                 else {
                     index_url = `--${this.highlightFlag("extra-index-url")}=https://pypi.anaconda.org/rapidsai-wheels-nightly/simple`;
                     var version = this.removeLeadingZeros("{{ site.data.releases.nightly.version }}")
-                    cuda_suffix = cuda_suffix + `>=${version}.0a0,<=${version}"`;
+                    cuda_suffix = cuda_suffix + `>=${version}.0a0,<=${version}`;
                     var libraryToPkg = (pkg) => {
                         pkg = pkg.toLowerCase();
-                        if (pkg === "cuspatial/cuproj") return ["\"cuspatial" + cuda_suffix, "\"cuproj" + cuda_suffix];
-                        if (pkg === "raft") return ["\"pylibraft" + cuda_suffix, "\"raft-dask" + cuda_suffix];
-                        if (pkg === "cuvs") return ["\"cuvs" + cuda_suffix, "\"pylibraft" + cuda_suffix];
-                        if (pkg === "cugraph/nx-cugraph") return ["\"cugraph" + cuda_suffix, "\"nx-cugraph" + cuda_suffix];
-                        return ["\"" + pkg + cuda_suffix];
+                        if (pkg === "cuspatial/cuproj") return ["cuspatial" + cuda_suffix, "cuproj" + cuda_suffix];
+                        if (pkg === "raft") return ["pylibraft" + cuda_suffix, "raft-dask" + cuda_suffix];
+                        if (pkg === "cuvs") return ["cuvs" + cuda_suffix, "pylibraft" + cuda_suffix];
+                        if (pkg === "cugraph/nx-cugraph") return ["cugraph" + cuda_suffix, "nx-cugraph" + cuda_suffix];
+                        return ["" + pkg + cuda_suffix];
```

Comment:
Is this empty string needed?
```suggestion
                        return [pkg + cuda_suffix];
```